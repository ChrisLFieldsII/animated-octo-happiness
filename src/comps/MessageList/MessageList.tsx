import React from 'react';

interface MessageListProps {
  messages: string[];
}

function MessageList({ messages }: MessageListProps) {
  return (
    <ul>
      {messages.map((msg, index) => (
        <li key={index}>{msg}</li>
      ))}
    </ul>
  );
}

export default MessageList;
