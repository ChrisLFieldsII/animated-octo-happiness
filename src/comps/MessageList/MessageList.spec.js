import React from 'react';
import renderer from 'react-test-renderer';
import MessageList from './MessageList';

describe('<MessageList />', () => {
  it('renders correctly', () => {
    const messages = ['Message 1', 'Message 2', 'Message 3'];

    const tree = renderer.create(<MessageList messages={messages} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
