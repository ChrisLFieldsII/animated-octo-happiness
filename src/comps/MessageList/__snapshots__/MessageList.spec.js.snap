// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MessageList /> renders correctly 1`] = `
<ul>
  <li>
    Message 1
  </li>
  <li>
    Message 2
  </li>
  <li>
    Message 3
  </li>
</ul>
`;
